connector_prestashop/models/prestashop_backend/common.py

```diff
@@ -120,6 +120,16 @@ def _select_versions(self):
         string='Importable sale order states',
         help="If valued only orders matching these states will be imported.",
     )
+    product_qty_field = fields.Selection(
+        selection=[
+            ('immediately_usable_qty', 'Immediately usable qty'),
+            ('qty_available', 'Qty available'),
```

Comment:
`virtual_available` is a good choice too (IMO way better choice than `qty_available`, I fail to see the use case where you want to sell the `qty_available` without consideration for the outgoing quantities). `immediately_usable_qty` is only possible if the module `stock_available_immediately` is installed, so should we add this option conditionally or be less restrictive and allow user the choose any field (including obsviously wrong fields) like in the magento connector: https://github.com/OCA/connector-magento/blob/66db2f4bcc9bff4abf75ce684a31d695bf295bc4/magentoerpconnect/magento_model.py#L153-L162

Reply:
Hi.. We could do PS dependent of this module, I think that incoming qty should not be exported to PS.. 
That you seem to add this dependency??

Reply:
I'd agree with adding the dependency as indeed it seems that immediately_usable_qty is the most sensible choice.

Reply:
`qty_available` is the current one and I kept it as I didn't won't to screw existing installations that deepends on it. Not sure if we should drop it now or if we should just log its deprecation.

+1 for dependency.